AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Event/Edit.cshtml

```diff
@@ -49,6 +49,11 @@
         <hr />
         <div class="form-horizontal col-lg-6 pull-left">
             <div asp-validation-summary="All" class="text-danger"></div>
+            @if (isEdit)
+            {
+                <input type="hidden" asp-for="Id" />
+                <input type="hidden" asp-for="ImageUrl" />
+            }
             <input asp-for="Id" type="hidden" />
```

Comment:
You have the hidden id input both inside and outside the if block here, so if in edit mode it would be duplicated currently. Looks like we can remove the line outside of the if block since we should only have an Id to maintain when editing an event.